modules/installation-mirror-repository.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Mirror the {product-title} image repository to use during cluster installation
+or upgrade.
+
+.Prerequisites
+
+* You prepared the bastion host to use in a disconnected installation.
+* You configured a mirror registry to use in your disconnected environment and
+can access the certificate and credentials that you configured.
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page and modified it to include authentication to your mirror repository.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the
+link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Quay.io]
+repository to determine the version of {product-title} that you want to install.
```

Comment:
Hmm.  Is this the UX we want?  I'd expect folks would want to use the most-recent release in their channel, which you can get with:

```console
$ curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.1' | jq -r '.nodes | sort_by(.version | sub(".*[.]"; "") | tonumber)[-1] | .version + "\t" + .payload'
4.1.16	quay.io/openshift-release-dev/ocp-release@sha256:61ed953962d43cae388cb3c544b4cac358d4675076c2fc0befb236209d5116f7
```

replacing `stable-4.1` with your favorite channel.  Or use `[]` instead of `[-1]` to get a list of all the choices.  And we could do that filtering in JavaScript if we wanted something live in the served docs.  Or are we not allowed to rely on JavaScript or something?

Lacking JavaScript, I'd expect there would be a customer-portal link we could point folks at with a list of supported versions.

Reply:
Do you have that customer portal link?

Reply:
https://access.redhat.com/downloads/content/290/